helm/configurations/nfs-provisioner-values.yaml

```diff
@@ -0,0 +1,8 @@
+# helm repo add stable https://charts.helm.sh/stable
+# helm install reana-dev-storage stable/nfs-server-provisioner -f nfs-provisioner-values.yaml
+# helm uninstall reana-dev-storage
+
+storageClass:
```

Comment:
@audrium Did this file appear here by mistake? (unrelated change...)

Reply:
yep, accidentally slipped through while I was rebasing